modules/appnexusBidAdapter.js

```diff
@@ -198,6 +198,7 @@ export const spec = {
         .replace(/<h1>(.*)<\/h1>/gm, '\n\n===== $1 =====\n\n') // Header H1
         .replace(/<h[2-6]>(.*)<\/h[2-6]>/gm, '\n\n*** $1 ***\n\n') // Headers
         .replace(/(<([^>]+)>)/igm, ''); // Remove any other tags
+      utils.logMessage("AppNexus Debug Auction Glossary: https://wiki.appnexus.com/x/qwmHAg");
```

Comment:
Can you change the double-quotes to single-quotes?  It's causing a linting error.

Reply:
yup. of course :-) Just pushed a fix.